specs/utility/time.table

```diff
@@ -1,26 +1,23 @@
 table_name("time")
-description("Track current date and time in the system.")
+description("Track current date and time in UTC.")
 schema([
-    Column("weekday", TEXT, "Current weekday in the system"),
-    Column("year", INTEGER, "Current year in the system"),
-    Column("month", INTEGER, "Current month in the system"),
-    Column("day", INTEGER, "Current day in the system"),
-    Column("hour", INTEGER, "Current hour in the system"),
-    Column("minutes", INTEGER, "Current minutes in the system"),
-    Column("seconds", INTEGER, "Current seconds in the system"),
-    Column("timezone", TEXT, "Current timezone in the system"),
-    Column("local_time", INTEGER, "Current local UNIX time in the system",
-        aliases=["localtime"]),
-    Column("local_timezone", TEXT, "Current local timezone in the system"),
-    Column("unix_time", INTEGER,
-        "Current UNIX time in the system, converted to UTC if --utc enabled"),
-    Column("timestamp", TEXT, "Current timestamp (log format) in the system"),
-    Column("datetime", TEXT, "Current date and time (ISO format) in the system",
+    Column("weekday", TEXT, "Current weekday in UTC"),
+    Column("year", INTEGER, "Current year in UTC"),
+    Column("month", INTEGER, "Current month in UTC),
```

Comment:
@theopolis This is missing an ending `"` in `"Current month in UTC` (that's also the reason the build is failing)